sema/check/sinks/html_sink.py

```
@@ -0,0 +1,83 @@
+
+    html_content += """
+        </tbody>
+    </table>
+</body>
+</html>
+    """
+
+    try:
+        with open(output_file, "w") as f:
+            f.write(html_content)
+    except IOError as e:
```

Ruff

Replace aliased errors with `OSError`

Replace `IOError` with builtin `OSError`

(UP024)